server.js

```diff
@@ -26,22 +26,23 @@ function Forecast(day) {
 app.get('/weather', async (request, response) => {
 
   
-  const city_name = request.query.city_name;
   const lat = request.query.lat;
   const lon = request.query.lon;
 
   const weatherAPI = `https://api.weatherbit.io/v2.0/forecast/daily?key=${process.env.REACT_APP_WEATHER_API_KEY}&land=en&lat=${lat}&lon=${lon}&days=5`
 
+  
 
+const firstClimate = await axios.get(weatherAPI);
 
+  
 
 
 
```

Comment:
Feedback here: Clean up line spacing when you have time on your next PR. 1 line between top level blocks is sufficient 👍